data/json/items/tool_armor.json

```diff
@@ -2211,6 +2211,51 @@
     },
     {
         "type" : "TOOL_ARMOR",
+        "id" : "bagpipes",
+        "name" : "bagpipes",
+        "name_plural": "bagpipes",
+        "category" : "tools",
+        "weight" : 2300,
+        "color" : "green",
+        "covers" : ["TORSO"],
```

Comment:
If the game differentiated between encumbrance granted when playing versus encumbrance granted when just wearing it that would be one thing, but as it stands it's a single value no matter whether you're actively playing it or just carrying it on your back. I'll bump up the penalties a little, but not a lot.